modules/learn/pages/clusters-and-availability/hard-failover.adoc

```diff
@@ -21,6 +21,29 @@ Hard failover can be initiated by means of:
 
 For further information on initiating hard failover, see xref:manage:manage-nodes/failover-hard.adoc[Perform Hard Failover].
 
+[#safe-and-unsafe]
+== Hard-Failover Modes: Safe and Unsafe
```

Comment:
I don't want to call "safe" hard failover as it sounds like contradiction in terms. I'd prefer to just call the whole thing "hard failover" and then refer to the fact that within the sphere of hard failover sometimes the admin is failing over a majority of nodes and those nodes can't be contacted. This is an extra wrinkle and we call it unsafe because we can't guarantee that another admin isn't doing the opposite failover from the other side of the network partition.